README.rst

```diff
@@ -33,20 +33,88 @@ absorption characteristics using a technique mimicking that presented in [AGHMMS
 finds a good solution for a non-negative matrix factorization by first transforming it to the problem of finding a convex
 hull for a set of points in a cloud.
 
+The lead devolper of InsightSoftwareConsortium/ITKColorNormalization is `Lee Newberg <https://github.com/Leengit)>`_.
+
+Installation for python
+-----------------------
+
+From the web page https://github.com/InsightSoftwareConsortium/ITKColorNormalization/actions, click on a recently
+successfully built workflow.  This takes you to a page of \"Build, test, package\" and a list of Artifacts, such as
+WindowWheel3.8, WindowWheel3.7, WindowWheel3.6, WindowWheel3.5, MacOSWheels, LinuxWheel38, LinuxWheel37, LinuxWheel36, and
+LinuxWheel35.  Click on the artifact that matches your operating system and version of python, to download a zip file.  Find
+the downloaded zip file, perhaps within your Downloads folder, and unzip it.  This gives you a file with a name like
+itk_spcn-0.1.0-cp38-cp38-manylinux1_x86_64.whl.
+
+In a terminal session, create a python virtual environment with a command like
+
+    python -m venv /tmp/venv
+
+Make your current terminal session use the virtual envrionment with a command like the following.  Note that the line begins
+with a single \".\" followed by a space.
+
+    . /tmp/venv/bin/activate
+
+Install the wheel in the virtual environment with a command like
+
+    /tmp/venv/bin/pip install itk_spcn-0.1.0-cp38-cp38-manylinux1_x86_64.whl
+
+You can stop using the virtual environment in your terminal session with the command
+
+    deactivate
+
+Usage
+-----
+
+If you haven't already, make your current terminal session use the previously created virtual envrionment with a command like
+the following.  Note that the line begins with a single \".\" followed by a space.
+
```

Comment:
Use triple backticks for code markup in Markdown.

Reply:
The issue title notwithstanding, the file is README.rst not README.md; I will commit the corresponding changes using rst markup.